hack/test-extended/builds/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# This scripts starts the OpenShift server with a default configuration.
+# The OpenShift Docker registry and router are installed.
+# It will start all 'default_*_test.go' test cases.
+
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../../..
+cd "${OS_ROOT}"
+
+source ${OS_ROOT}/hack/util.sh
+source ${OS_ROOT}/hack/common.sh
+
+
+set -e
+ensure_ginkgo_or_die
+set +e
+
+os::build::extended
+
+ensure_iptables_or_die
+
+function cleanup()
+{
+	out=$?
+	cleanup_openshift
+	echo "[INFO] Exiting"
+	exit $out
+}
+
+echo "[INFO] Starting 'builds' extended tests"
+
+trap "exit" INT TERM
+trap "cleanup" EXIT
+
+export TMPDIR="${TMPDIR:-"/tmp"}"
+export BASETMPDIR="${TMPDIR}/openshift-extended-tests/builds"
+setup_env_vars
+reset_tmp_dir 
+configure_os_server
+start_os_server
+
+install_registry
+wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 -t "{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
```

Comment:
> Do we want to wrap this with a nice name for readability or shove it into install_registry?

We don't want to combine it because that would limit the parallelizability of future scripters (install all 10 of these things and then starting waiting for completion).

I'm not against a new helper for it.